src/index.js

```diff
@@ -317,6 +321,20 @@ class SentryCliPlugin {
         return this.cli.releases.new(release);
       })
       .then(() => this.cli.releases.uploadSourceMaps(release, this.options))
+      .then(() => {
+        const { commit, previousCommit, repo, auto } = this.options;
+
+        if (repo && ((commit && previousCommit) || auto)) {
+          return this.cli.releases.setCommits(release, {
+            commit,
+            previousCommit,
+            repo,
+            auto,
+          });
+        }
+
+        return undefined;
```

Comment:
This line is redundant, as non-return value for `then` handler is already `undefined` anyway

Reply:
Fixed